packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -2140,4 +2135,68 @@ QUnit.module('aria accessibility', () => {
             });
         });
     });
+
+    QUnit.module('aria-autocomplete', () => {
+        const defaultConfigurationFalse = {
+            acceptCustomValue: false,
+            searchEnabled: false,
+        };
+
+        const defaultConfigurationTrue = {
+            acceptCustomValue: true,
+            searchEnabled: true,
+        };
+
+        const configurations = [
+            [{
+                acceptCustomValue: true,
+                searchEnabled: false,
+            }, 'list'],
```

Comment:
when `searchEnabled=false` no list is shown on typing. Are u sure we should have "list" value here?
Let's discuss in personal.